.github/actions/changelogs-generator/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Generate Changelogs
+description: 'Bump Version and Generate Changelogs'
+inputs:
+  repo_name:
+    description: 'Name of the repo in which action is called'
+    required: true
+  version:
+    description: 'New version number'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        repository: ${{ inputs.repo_name }}
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install package
+      shell: bash
+      run: pip install python-semantic-release
+
+    - name: Git setup
+      shell: bash
```

Comment:
BTW, you can skip `shell: bash` in each step by using this at the top level of the yml file:
```yaml
defaults:
  run:
    shell: bash
```

Reply:
We can specify shell at the top for workflow but for actions, we have to add it in every step